gadget/update.go

```diff
@@ -86,6 +87,13 @@ type ContentObserver interface {
 		targetRootDir, relativeTargetPath string, dataChange *ContentChange) (bool, error)
 }
 
+type ContentInstallObserver interface {
```

Comment:
Could we rather have something like `SystemInstallObserver` interface defined in gadget/install/install.go, which would like like so:

```
type SystemInstallObserver interface {
	gadget.ContentObserver
	// ChosenEncryptionKey stores the encrypted data partition key to be sealed
	// at the end of the installation process.
	ChosenEncryptionKey(secboot.EncryptionKey)
}
```

Reply:
Will move the interface definition to gadget/install as suggested.